### PR TITLE
Feature/general improvements

### PR DESCRIPTION
**Pull request by 0xCA5A**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ The list is not complete.
 
 
 ## jbert Application Release Notes
```
Comment (rake5k) [inline README.md]: Maybe we want to move the release notes into a CHANGELOG.md?

File: ansible/deploy.yml [hunk 2]
```diff
@@ -43,6 +43,15 @@
...
         state: absent
 
+    - name: Copy application.conf if existing
+      copy:
+        src: 'application.conf'
+        dest: '/usr/share/jbert/conf/application.conf'
+        owner: '{{ jbert_application_user}}'
+        follow: true
+        force: true
+      ignore_errors: yes
```
Comment (rake5k) [inline ansible/deploy.yml]: Why do we need to ignore errors here? I think this should not fail, ideally.

File: application/conf/reference.conf [hunk 3]
```diff
@@ -20,7 +20,7 @@ play {
 backend {
   halMockEnabled = false
   mpd {
-    server = "localhost"
+    server = "127.0.0.1"
```
Comment (rake5k) [inline application/conf/reference.conf]: Is there a situation when `localhost` does not work?

File: application/conf/logback.xml [hunk 4]
```diff
@@ -1,35 +1,30 @@
...
-  <logger name="play" level="INFO" />
-  <logger name="application" level="DEBUG" />
-
-  <root level="WARN">
-    <appender-ref ref="ASYNCFILE" />
-    <appender-ref ref="ASYNCSTDOUT" />
-  </root>
+    <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel"/>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```
Comment (rake5k) [inline application/conf/logback.xml:6]: No more log file on the jbert?

**Review by rake5k — APPROVED** [review]
Looks good to me!

File: README.md (hunk 1 above)
Comment (0xCA5A) [reply README.md]: yes!

File: ansible/deploy.yml (hunk 2 above)
Comment (0xCA5A) [reply ansible/deploy.yml]: Thought it was a good idea to make deployment possible without config file - now its not so good as i see now.

File: application/conf/reference.conf (hunk 3 above)
Comment (0xCA5A) [reply application/conf/reference.conf]: There was some a MPD connection issue once... ill kick this out!

File: application/conf/logback.xml (hunk 4 above)
Comment (0xCA5A) [reply application/conf/logback.xml:6]: No, journalctl -f is good enough for now... as soon as we need it, it comes back in.
The goal is to keep IO low - might not be the right place to start but its a start...

**Pull request merged**